FishjamClient/src/main/java/org/membraneframework/rtc/dagger/MembraneRTCComponent.kt

```diff
@@ -0,0 +1,18 @@
+package org.membraneframework.rtc.dagger
+
+import android.content.Context
+import org.membraneframework.rtc.InternalMembraneRTC
+import org.webrtc.EglBase
+import org.webrtc.audio.AudioDeviceModule
+
+internal interface MembraneRTCComponent {
```

Comment:
Is this file still required?

Reply:
No, merge from main pulled this file. Deleted.